pyro/distributions/transforms/spline.py

```diff
@@ -254,6 +254,7 @@ def _monotonic_rational_spline(
             c = -input_delta * (inputs - input_cumheights)
 
             discriminant = b.pow(2) - 4 * a * c
+            discriminant[outside_interval_mask] = 0 # added to make sure outside_interval input can be reversed as identity.
```

Comment:
I think you'll need two spaces before the comment, and possibly to move  the line comment to a separate line to satisfy max line length requirements.  You can locally run `make lint` to see lint errors.

Also you might need to use a non-inplace version like
```python
discriminant = discriminant.masked_fill(outside_interval_mask, 0)
```

Reply:
Thanks for your comment. I am new to pulling a request. So in this case, it is because of my non-standard comment that my request fails, right?

Reply:
I believe so, we use the `black` automatic style checker to ensure all code follows the same style.  This helps everyone collaborate by making the code more uniform.

Reply:
I get it. Thanks. I will edit it soon

Reply:
I have re-pulled it. Please check it.